tutorials/auto_inertia_calculation.md

```diff
@@ -0,0 +1,163 @@
+\page auto_inertia_calculation Automatic Inertia Calculation for Links
+
+## Automatic Inertia Calculation for SDFormat Links
+
+This feature enables automatic calculation for the Moments of Inertia, Mass, and
+Inertial Pose (Center of Mass pose) of a link described using SDFormat. The following
+geometry types are currently supported for this feature:
+ * Box
+ * Capsule
+ * Cylinder
+ * Ellipsoid
+ * Sphere
+ * Mesh
+
+Using this feature, a user can easily set up an accurate simulation with physically
+plausible inertial values for a link. This also removes the dependency on manual calculations
+or 3rd-party mesh processing software which can help lower the barrier of entry for beginners.
+
+This tutorial will focus on how this feature can be enabled and how the
+inertia values for a link can be configured. Some limitations and recommendations
+would also be discussed along the way that would allow users to more mindfully utilize this feature.
+
+## Basic Overview
+
+This feature introduced a new `auto` attribute for the `<inertial>` tag which can be set
+ to enable or disable the automatic calculations. It'll be set to false by default.
+
+```xml
+<inertial auto="true" />
+```
+
+In case, `auto` is set to true, the constituent **collision geometries** of the link are
+considered for the calculations. The `<density>` tag can be used to specify
+the mass density value of the collision in kg/m^3. The density of water (1000 kg/m^3) is
+utilized as the default value:
+
+```xml
+<collision name="collision">
+  <density>2710.0</density>
+  <geometry>
+    <box>
+      <size>1 1 1</size>
+    </box>
+  </geometry>
+</collision>
+```
+
+In case of multiple collision geometries in a link, a user is free to provide different
+density values for each and the inertia values from each would be aggregated to calculate
+the final inertia of the link. However, if there are no collisions present,
+an `ELEMENT_MISSING` error would be thrown.
+
+It is **important** to note here that if `auto` is set to `true` and the user has
+still provided values through the `<mass>`, `<pose>` and `<inertia>` tags, they
+would be **overwritten** by the automatically computed values.
+
+>**Note:** Use SDF Spec version 1.11 or greater to utilize the new tags and attributes
+of this feature.

```

Comment:
This is needed for proper formatting:

```suggestion
> **Note:** Use SDF Spec version 1.11 or greater to utilize the new tags and attributes
> of this feature.

```

btw, you can test this locally by running `make doc` in gz-sim build directory and opening `doxygen/index.html`.

Reply:
Thank you for the note. I have updated the formatting in [6217798](https://github.com/gazebosim/gz-sim/pull/2119/commits/6217798a922afdfbbd7c690803b1774c041502b7)